docs/attributes-registry/agent.md

```diff
@@ -0,0 +1,21 @@
+<!--- Hugo front matter used to generate the website version of this page:
+--->
+
+# Agent
+
+## Agent Attributes
+
+<!-- semconv registry.agent(omit_requirement_level) -->
+| Attribute  | Type | Description  | Examples  | Stability |
+|---|---|---|---|---|
+| `agent.distro` | string | Agent distribution. [1] | `github.com/signalfx/splunk-otel-collector`; `github.com/aws-observability/aws-otel-collector` | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
```

Comment:
I am always unsure what the stance is on semantic conventions regarding whether to be general vs specific. Other kinds of software can also have 'distros', is that an argument to make this more general? Is it okay to have a specific and a general convention?

Reply:
This works for me, I still have the general question but this is something that semconv maintainers can bring up if needed